hedgehog/test/Test/Hedgehog/Seed.hs

```diff
@@ -0,0 +1,84 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Test.Hedgehog.Seed (
+    tests
+  ) where
+
+import           Data.Foldable (for_)
+
+import           Hedgehog
+import qualified Hedgehog.Internal.Seed as Seed
+
+data Assert =
+  Assert {
+      expected :: !Seed
+    , actual   :: !Seed
```

Comment:
Ugh! I forgot we're strict with no alignment anywhere (look at line 14). Happy to clean this up if you wish. /cc @jystic @thumphries